scripts/THALES_migration/stakingEscrowMigration/executeStakingAndEscrowMigration.js

```diff
@@ -66,23 +70,36 @@ async function executeStakingAndEscrowMigration() {
 			continue;
 		}
 
+		if (processedWallets.includes(migratedStakerOrEscrower.wallet)) {
+			console.log('Skipping ' + migratedStakerOrEscrower.wallet + ' as it was already processed');
+			continue;
+		}
 		//send directly if not a staker
-		console.log('Processing migratedStakerOrEscrower ' + migratedStakerOrEscrower);
+		console.log('Processing migratedStakerOrEscrower ' + migratedStakerOrEscrower.wallet);
 		if (migratedStakerOrEscrower.totalStaked == 0) {
-			await thales.transfer(
+			console.log('Sending THALES directly to  ' + migratedStakerOrEscrower.wallet);
+			let tx = await thales.transfer(
 				migratedStakerOrEscrower.wallet,
-				w3utils.toWei(migratedStakerOrEscrower.totalEscrowed / 1e18 + '')
+				migratedStakerOrEscrower.totalEscrowed + ''
 			);
+			tx.wait().then(e => {
+				txLog(tx, 'thales: transfer ' + migratedStakerOrEscrower.wallet);
+			});
 		}
 		//else put to staked and send $10 ETH if the staker has none
 		else {
-			let escrowed = Big(migratedStakerOrEscrower.totalEscrowed / 1e18);
+			let escrowed;
+			if (migratedStakerOrEscrower.totalEscrowed === undefined) {
+				escrowed = BigNumber.from(0);
+			} else {
+				escrowed = BigNumber.from(migratedStakerOrEscrower.totalEscrowed);
+			}
 			console.log('Escrowed is ' + escrowed);
-			let staked = Big(migratedStakerOrEscrower.totalStaked / 1e18);
+			let staked = BigNumber.from(migratedStakerOrEscrower.totalStaked);
```

Comment:
maybe another check for `if(migratedStakerOrEscrower.totalStaked === undefined)`.

Reply:
not possible as we always prepare that in the prepareStakingAndEscrowMigrationData.js